packages/components/src/mobile/image/image-editing-button.native.js

```diff
@@ -27,7 +33,12 @@ const ImageEditingButton = ( {
 			source={ { uri: url } }
 			openReplaceMediaOptions={ openMediaOptions }
 			render={ ( { open, mediaOptions } ) => (
-				<TouchableWithoutFeedback onPress={ open }>
+				<TouchableWithoutFeedback
```

Comment:
The changes in this PR may not actually have the desired impact for addressing wordpress-mobile/gutenberg-mobile#2699, for a couple of reasons I have discovered.

1. This edit image button was rendered _beneath_ the image itself, which means it was not reachable when using voice over. I have [relocated it to address this](https://github.com/WordPress/gutenberg/pull/25810/commits/b07e0bccdefe9b3f778d7cdfa9477f270f578509).
    <details>
    <summary>Unreachable edit button</summary>
    
    ![unreachable-edit-image button](https://user-images.githubusercontent.com/438664/107584498-2e53bf80-6bc2-11eb-85e4-6ca22b9610f6.PNG)
    
    </details>
2. I believe wordpress-mobile/gutenberg-mobile#2699 was describing the lack of a label for the _add media button_, which is displayed when no cover media has been set. I have [added a label for this button](https://github.com/WordPress/gutenberg/pull/25810/commits/e96e1d40bd7ba01bb99b402ab825b7c06cf4f954).

    <details>
    <summary>Label-less add button</summary>
    
    ![label-less-add-media-button](https://user-images.githubusercontent.com/438664/107584531-3c094500-6bc2-11eb-8ae7-2cb94bdf1454.PNG)
    
    https://user-images.githubusercontent.com/438664/107584541-3e6b9f00-6bc2-11eb-89cd-183bbf6d65ee.MP4
    
    </details>